app/serializers/discussion_serializer.rb

```diff
@@ -38,6 +39,10 @@ def reader
     @reader ||= scope[:reader_cache].get_for(object) if scope[:reader_cache]
   end
 
+  def edited
+    object.versions.where(event: 'update').count > 1
```

Comment:
Ooh, we want to store this on the discussion once it's updated I reckon; this query is too expensive to call with every discussion serialized.

I think the easiest way to do this might be to put a counter cache on the number of versions, and check that

```
#discussion.rb
define_counter_cache :versions_count { |discussion| discussion.versions.where(event: :update).count }
```

```
# discussion_serializer.rb
def edited
  object.versions_count > 1
end
```

Something like that?